server/cmake/modules/FindLibClang.cmake

```diff
@@ -16,7 +16,7 @@
 # most recent versions come first
 # http://llvm.org/apt/
 set(LIBCLANG_KNOWN_LLVM_VERSIONS 5.0.0 5.0
```

Comment:
-- C++11 compiler option(s): -std=c++11
    -- Found:  /usr/lib/llvm/4/lib64/libclang.so.4.0
    -- Found LibClang: /usr/lib64/llvm/4/lib64/libclang.so  
    -- Detecting libclang builtin headers directory
    -- Found:  /usr/lib/llvm/4/lib64/libclang.so.4.0
    -- Detecting libclang builtin headers directory -- success
    -- Irony package version is '0.2.2-cvs'

Looks like it's finding the library but not the headers?